ExtraEnemyCustomization/Customizations/EnemyAbilities/Abilities/EMP/Handlers/EMPGunSightHandler.cs

```diff
@@ -0,0 +1,41 @@
+﻿using EECustom.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+using UnityEngine;
+
+namespace EECustom.Customizations.EnemyAbilities.Abilities.EMP.Handlers
+{
+    public class EMPGunSightHandler : EMPHandlerBase
+    {
+        public GameObject _sightPicture;
+        const string _regex = "Sight_[0-9]*_[G-g]lass";
```

Comment:
Protection definition is missing, inconsistent with rest of codebase.
```suggestion
        private const string _regex = @"Sight_[0-9]*_[G-g]lass";
```

Just as a note: I would personally inline this at L17